dash-renderer/setup.py

```diff
@@ -1,16 +1,16 @@
 from setuptools import setup
 
-version = {}
-exec(open('dash_renderer/version.py').read(), version)  # pylint: disable=exec-used
+with open('VERSION.txt', 'r') as fp:
```

Comment:
this leaves the door for auto bump version if we want in the future, and also better than the `exec` version in my opinion

Reply:
Could we derive the version from the package.json instead, one source of truth!

Reply:
one more step further towards a JS project :)